samples/step-by-step/sample.md

```diff
@@ -20,29 +19,50 @@ The goal of this step-by-step sample is to show how to build a simple system tha
 
 Completing these steps will serve as an introduction to many important NServiceBus concepts:
 
-* Basic project setup
-* [Self-hosting NServiceBus](/nservicebus/hosting/) within a console application
-* [Defining commands and events](/nservicebus/messaging/messages-events-commands.md)
-* [Sending commands](/nservicebus/messaging/send-a-message.md)
-* [Handling messages](/nservicebus/handlers/)
-* [Publishing events](/nservicebus/messaging/publish-subscribe/publish-handle-event.md)
-* [Logging](/nservicebus/logging/)
+ * Basic project setup
+ * [Self-hosting NServiceBus](/nservicebus/hosting/) within a console application
+ * [Defining commands and events](/nservicebus/messaging/messages-events-commands.md)
+ * [Sending commands](/nservicebus/messaging/send-a-message.md)
+ * [Handling messages](/nservicebus/handlers/)
+ * [Publishing events](/nservicebus/messaging/publish-subscribe/publish-handle-event.md)
+ * [Logging](/nservicebus/logging/)
 
 
 ## Prerequisites
 
-* [.NET Framework Version 4.5.2](https://www.microsoft.com/en-au/download/details.aspx?id=42642)
-* Microsoft Message Queueing (MSMQ) must be [properly installed and configured](/nservicebus/msmq/). The easiest way to do this is using the [Platform Installer](/platform/installer/) tool.
+
+### .NET Framework Version 4.5.2
+
+[Download the .NET Framework Version 4.5.2](https://www.microsoft.com/en-au/download/details.aspx?id=42642) and install it.
+
+
+### MSMQ
+
+Microsoft Message Queueing (MSMQ) must be properly installed and configured. The easiest way to do this is with the appropriate command below from the command line. Alternatively, it can be installed via the Windows GUI following the [MSMQ guide](/nservicebus/msmq/).
+
+
+#### Enable MSMQ on Windows 7
```

Comment:
as in we already link to a pge that shows them how to install msmq. lets not repeat it on this page.